Disable the PowerPC counter, because UnscaledCycleClock::Frequency()
is not implemented on OpenBSD.  Avoids error in unscaledcycleclock.cc,
#error Must implement UnscaledCycleClock::Frequency()

Index: absl/base/internal/unscaledcycleclock.h
--- absl/base/internal/unscaledcycleclock.h.orig
+++ absl/base/internal/unscaledcycleclock.h
@@ -46,7 +46,7 @@
 
 // The following platforms have an implementation of a hardware counter.
 #if defined(__i386__) || defined(__x86_64__) || defined(__aarch64__) || \
-    defined(__powerpc__) || defined(__ppc__) || defined(__riscv) ||     \
+    /*defined(__powerpc__) || defined(__ppc__) ||*/ defined(__riscv) ||     \
     defined(_M_IX86) || defined(_M_X64)
 #define ABSL_HAVE_UNSCALED_CYCLECLOCK_IMPLEMENTATION 1
 #else
